bluebird.d.ts

```diff
@@ -151,8 +151,8 @@ declare class Bluebird<R> implements Bluebird.Thenable<R>, Bluebird.Inspection<R
    * });
    * </code>
    */
-  // TODO find way to fix get()
-  // get<U>(propertyName: string): Bluebird<U>;
+  // TODO: explain original comment from DT: TODO find way to fix get()
```

Comment:
Ok, thank you for your explanation. Are these subtypes ready yet ?
Anyway, it's better to use a sub-optimal declaration than no declaration at all.